### PR TITLE
FM bugfix: Always initialize idle timer 

### PR DESCRIPTION
**Pull request by connorwstein**

### CODE REVIEW

**Review by PiotrTrzpil — DISMISSED** [review]
LGTM - @spooktheducks what do you think?

File: core/services/fluxmonitor/flux_monitor.go [hunk 1]
```diff
@@ -434,6 +434,10 @@ func NewPollingDeviationChecker(
 	fetcher Fetcher,
 	minSubmission, maxSubmission *big.Int,
 ) (*PollingDeviationChecker, error) {
+	var idleTimer = utils.NewResettableTimer()
+	if !initr.IdleTimer.Disabled {
+		idleTimer.Reset(initr.IdleTimer.Duration.Duration())
+	}
```
Comment (samsondav) [inline core/services/fluxmonitor/flux_monitor.go:440]: Does fmv2 have this problem?

Comment (connorwstein) [reply core/services/fluxmonitor/flux_monitor.go:440]: yah and I also fixed it in this PR

File: core/services/fluxmonitor/flux_monitor_test.go [hunk 2]
```diff
@@ -1463,6 +1463,54 @@ func TestFluxMonitor_PollingDeviationChecker_HandlesNilLogs(t *testing.T) {
 	})
 }
 
+func TestFluxMonitor_IdleTimer(t *testing.T) {
```
Comment (samsondav) [inline core/services/fluxmonitor/flux_monitor_test.go:1466]: Can we have a test on the v2 fix also?

**Review by samsondav — CHANGES_REQUESTED** [review]

File: core/services/fluxmonitor/flux_monitor_test.go (hunk 2 above)
Comment (connorwstein) [reply core/services/fluxmonitor/flux_monitor_test.go:1466]: there is one - thats what https://github.com/smartcontractkit/chainlink/pull/4397/files#diff-79e16dc4b7fdbc22de88a5d4da5a454d4fb0adb9e4e7c0ac73d669b877608e51R115 is

**Review by samsondav — APPROVED** [review]

**Pull request merged**